src/main/scala/coreplex/Configs.scala

```diff
@@ -27,6 +28,7 @@ class BaseCoreplexConfig extends Config ((site, here, up) => {
   case ErrorParams => ErrorParams(Seq(AddressSet(0x3000, 0xfff)))
   case BootROMParams => BootROMParams(contentFileName = "./bootrom/bootrom.img")
   case DebugModuleParams => DefaultDebugModuleParams(site(XLen))
+  case PropertyLibrary => (new BasePropertyLibrary[BasePropertyParameters])
```

Comment:
I wouldn't add a default here until we come up with a specific default behavior applied to instances of `cover(...)` in rocket-chip. See below...